src/vm/CMakeLists.txt

```diff
@@ -331,6 +331,7 @@ set(VM_SOURCES_WKS
     eventpipemetadatagenerator.cpp
     eventpipeprotocolhelper.cpp
     eventpipeprovider.cpp
+    eventpipethread.cpp
```

Comment:
Do you need to add the `eventpipethread.h` below (eventpipe*.h headers are there)?